UI/CardPileUISetter.gd

```diff
@@ -22,3 +22,9 @@ func _on_card_counts_updated() -> void:
 
 func _get_count() -> int:
 	return CardManager.card_container.get_draw_pile_size()
+
+func _pressed() -> void:
+
+	var uiPile = cardUI.instantiate()
+	uiPile.populate(get_name())
+	$"..".add_child(uiPile)
```

Comment:
maybe a `var parent = $".."` then `parent.add_child` would be a bit less cryptic. Not sure that syntax is possible though